testdata/descriptions.yml

```diff
@@ -2879,7 +2879,7 @@ parameters:
 # decision_function_shape, break_ties ignored because we only have binary problems so far    
 ---
 
-name: SKLEARN_LinearDiscriminantAnalysis_SVD
+name: SKLEARN_LinearDiscriminantAnalysisSVD
```

Comment:
please undo, the second underscores where intentional. Pattern: 
framework_algo_hyperparameterset